tests/python/pants_test/option/test_options_integration.py

```diff
@@ -275,3 +277,19 @@ def test_pants_ignore_option(self):
       self.assertIn("pants_ignore = ['.*/', '/dist/', 'some/random/dir'] (from CONFIG in {})"
                     .format(config_path),
                     pants_run.stdout_data)
+
+  def test_pants_symlink_workdirs(self):
+    with temporary_dir('/tmp') as tmp_dir:
+      symlink_workdir = f'{get_buildroot()}/.pants.d'
+      physical_workdir_base = f'{tmp_dir}/workdirs'
+
+      pants_run = self.run_pants_with_workdir(
+        [f'--pants-physical-workdir-base={physical_workdir_base}', 'help'], symlink_workdir)
+      self.assert_success(pants_run)
```

Comment:
After this call, it would be good confirm that the workdir is actually a symlink.